src/components/FormProduct/index.jsx

```diff
@@ -14,88 +14,139 @@ export const FormProduct = () => {
+        if (isProductDuplicated(products, item)) {
+          setError(true);
+          setErrorMsg('The item is already on the list');
+        } else {
+          const editedProduct = {
+            ...product,
+            nextPurchase: Number(nextPurchase),
+            creationDate: new Date(),
+          };
+          addProduct(editedProduct);
+          setProduct({ ...initialStateProduct });
+          setError(false);
+          setErrorMsg('');
+          setSuccessMsg(true);
+        }
+      } else {
+        setError(true);
+        setErrorMsg('Next Purchase is required');
+        setSuccessMsg(false);
+      }
+    } else {
+      setError(true);
+      setErrorMsg('Name is required');
+      setSuccessMsg(false);
+    }
   };
 
   return (
-    <div>
-      {error && <p>{error}</p>}
-      <form onSubmit={handleSubmitProduct}>
+    <div className="d-flex justify-content-center container-form w-100">
+      <form onSubmit={handleSubmitProduct} className="w-100 p-2" noValidate>
         <div>
-          <label>Item name: </label>
+          <label className="form-text" htmlFor="Item-Name">
+            Item name:
+          </label>
           <input
-            onChange={handleInputProduct}
+            className="form-control"
+            id="Item-Name"
+            onChange={handleNameInput}
             type="text"
-            placeholder="Add A Product"
+            placeholder="Add a Product"
             autoComplete="off"
             required="required"
             name="item"
             value={product.item}
           />
         </div>
-        <div>
-          <label>How soon will you buy this again?</label>
-          <br />
-          <input
-            onChange={handleInputProduct}
-            type="radio"
-            required="required"
-            name="nextPurchase"
-            value={6}
-          />
-          Soon
-          <br />
-          <input
-            onChange={handleInputProduct}
-            type="radio"
-            required="required"
-            name="nextPurchase"
-            value={14}
-          />
-          Kind Of Soon
-          <br />
-          <input
-            onChange={handleInputProduct}
-            type="radio"
-            required="required"
-            name="nextPurchase"
-            value={31}
-          />
-          Not Soon
-          <br />
+        <div className="mt-3 mb-3">
+          <p className="form-text">How soon will you buy this again?</p>
+          <div className="radio-buttons" role="radiogroup">
+            <input
+              onChange={handleRadioNextPurchase}
+              type="radio"
+              required="required"
+              name="options"
+              value={6}
+              id="option-soon"
+              checked={product.nextPurchase === '6' ? true : false}
+            />
+            <label className="btn btn-soon" htmlFor="option-soon">
+              <i class="bi bi-stopwatch-fill" style={{ fontSize: 24 }}></i>
```

Comment:
I am not sure of how valuable the selection of the icons is, they don't seem to communicate the desired message, but I look forward to hearing more about the process selection during our weekly sync

Reply:
Thanks! OK, in our weekly sync we can talk about it.